.github/Issue_Template/Other.yml

```
@@ -0,0 +1,23 @@
+name: Other
+description: Use this for any other issues. Please do NOT create blank issues
```

```suggestion
description: File any other issues.
```
Black issues cannot be created due to `required: true` value. I think we don't need to state that.